packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -118,37 +165,16 @@ function DataViews< Item >( {
 		( filters || [] ).some( ( filter ) => filter.isPrimary )
 	);
 
-	const defaultUI = (
-		<>
-			<HStack
-				alignment="top"
-				justify="space-between"
-				className="dataviews__view-actions"
-				spacing={ 1 }
-			>
-				<HStack
-					justify="start"
-					expanded={ false }
-					className="dataviews__search"
-				>
-					{ search && <DataViewsSearch label={ searchLabel } /> }
-					<FiltersToggle />
-				</HStack>
-				<HStack
-					spacing={ 1 }
-					expanded={ false }
-					style={ { flexShrink: 0 } }
-				>
-					<DataViewsViewConfig />
-					{ header }
-				</HStack>
-			</HStack>
-			{ isShowingFilter && (
-				<DataViewsFilters className="dataviews-filters__container" />
-			) }
-			<DataViewsLayout />
-			<DataViewsFooter />
-		</>
+	const defaultUI = useMemo(
```

Comment:
I don't think we need to memoize this. `createElement` is not expensive on its own. I'm guessing the idea was to avoid all the nested createElements.